src/main/createStore.test.ts

```diff
@@ -2,7 +2,7 @@ import 'jest-extended';
 
 import { createStore } from './createStore';
 import {
-	SiteDataBySiteID,
+	CachedImageDataBySiteID,
```

Comment:
Thoughts on swapping these back to `SiteDataBySiteID`? This was renamed recent from `CachedImageDataBySiteID` -> `SiteDataBySiteID`; I thought the latter was a little cleaner. TBH, I can go either way. Mostly just wanted to make sure this was intentional! 😄